README.md

```diff
@@ -32,10 +32,16 @@ npm i --save @ionos-cloud/sdk-nodejs
 ```
 
 ## Usage
+Import the SDK using:
 
-```javscript
-const sdk = require('@ionos-cloud/sdk-nodejs')
+```const sdk = require('@ionos-cloud/sdk-nodejs')```
+
+Or, if the import is done from an ES module, use:
 
+```import * as sdk from '@ionos-cloud/sdk-nodejs';```
+
+Usage example:
+```javscript

```

Comment:
Should probably be `javascript`.

Reply:
@DanielRuf done in https://github.com/ionos-cloud/sdk-nodejs/pull/9/commits/6ee965d01a63236d59dd737f88a78cd94f0d22a1.